src/_profile-photos.scss

```diff
@@ -45,10 +45,26 @@
   & > .profile-photo-large:not(:first-child) {
     margin-left: -12px;
   }
+
   & > .profile-photo-medium:not(:first-child) {
     margin-left: -8px;
   }
+
   & > .profile-photo-small:not(:first-child) {
     margin-left: -4px;
   }
+
+  *:not(:first-child) {
+    .profile-photo-large {
+      margin-left: -12px;
+    }
+
+    .profile-photo-medium {
+      margin-left: -12px;
+    }
+
+    .profile-photo-small {
+      margin-left: -12px;
+    }
+  }
```

Comment:
The output looks correct, but I'm not sure why 😅
Shouldn't these be -12, -8, and -4, respectively?

Reply:
Oops, good catch